src/mv3/api.ts

```diff
@@ -20,12 +20,15 @@
 import { type Tabs } from "webextension-polyfill";
 import { once } from "lodash";
 
-export const isMV3 = once(
-  (): boolean =>
-    // Use optional chaining in case the chrome runtime is not available:
-    // https://github.com/pixiebrix/pixiebrix-extension/issues/8273
-    chrome.runtime?.getManifest().manifest_version === 3,
-);
+export const isMV3 = once((): boolean => {
+  if (!chrome.runtime?.getManifest) {
+    return false;
+  }
+
+  // Use optional chaining in case the chrome runtime is not available:
+  // https://github.com/pixiebrix/pixiebrix-extension/issues/8273
+  return chrome.runtime.getManifest().manifest_version === 3;
```

Comment:
I'm pretty sure you can also just keep chaining the null-safe access as before, if you want:
```suggestion
  if (!chrome.runtime?.getManifest) {
    return false;
  }

  // Use optional chaining in case the chrome runtime is not available:
  // https://github.com/pixiebrix/pixiebrix-extension/issues/8273
  return chrome.runtime?.getManifest?.()?.manifest_version === 3;
```

Reply:
Thanks ben, I actually didn't remember that you could put the double parentheses after the null operator.